### DISCUSSION TITLE
Abogabot

### DISCUSSION THREAD

**Discussion by DiegoCorchuelo** [Launch-X-Latam/MisionFrontEnd#245] [General]
Hola, He completado la practica y acepto opiniones o consejos sobre ella.
Aquí les dejo el enlace a mi repositorio.
https://github.com/DiegoCorchuelo/Abogabot

**Comment by lopez-juandavid** [comment]
Me aparece que tu repositorio esta vacío!

**Reply by DiegoCorchuelo** [reply]
Hola Juan, Si estaba haciendo unos cambios pero ya lo puedes ver.
